src/go/install.sh

```diff
@@ -219,8 +219,18 @@ if [ "${INSTALL_GO_TOOLS}" = "true" ]; then
 
     # Move Go tools into path and clean up
     mv /tmp/gotools/bin/* ${TARGET_GOPATH}/bin/
-
     rm -rf /tmp/gotools
+
+    # Install golangci-lint from precompiled binares
+    if [ "$GOLANGCILINT_VERSION" = "latest" ] || [ "$GOLANGCILINT_VERSION" = "" ]; then
+        echo "Installing golangci-lint latest..."
+        curl -fsSL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | \
+            sh -s -- -b "${TARGET_GOPATH}/bin"
```

Comment:
`TARGET_GOROOT: /usr/local/go` whereas `TARGET_GOPATH: /go`

```suggestion
            sh -s -- -b "${TARGET_GOROOT}/bin"
```

Reply:
Why GOROOT? The Go tools are copied into GOPATH: https://github.com/devcontainers/features/pull/238/files#diff-8d791f705e135e8d78a36c8218fd50d0d199af1f72c8df7721ad34f8865b627fR221

Reply:
I thought the copying over happens later in the code. But doesn't look like it.

Feel free to ignore the comment.